x-pack/test/alerting_api_integration/security_and_spaces/group3/tests/alerting/clone.ts

```diff
@@ -173,6 +173,7 @@ export default function createAlertTests({ getService }: FtrProviderContext) {
                   },
                   outcome: 'succeeded',
                   outcome_msg: null,
+                  outcome_order: 0,
```

Comment:
We should add tests for 'warning' and 'failed' to ensure the `outcome_order` is expected.  I think that can just be added to any other test that receives those outcomes.

Reply:
Not sure what tests you mean here. I've added more test cases to the migrations suite, and haven't found any explicit integrational tests for the `lastRun` or `outcome` fields. This file is the only place in ` 
x-pack/test/alerting_api_integration` besides migrations where execution outcome is surfaced.

Reply:
yes, we will add specific test around that in the near future. We will use the same one for `executionStatus` if they exist

Reply:
They do exist! So here a ticket https://github.com/elastic/kibana/issues/150487 to add them for `last_run`